modules/UI/videolayout/SmallVideo.js

```diff
@@ -319,7 +325,9 @@ SmallVideo.prototype.createModeratorIndicatorElement = function () {
  * Removes the element indicating the moderator(owner) of the conference.
  */
 SmallVideo.prototype.removeModeratorIndicatorElement = function () {
-    $('#' + this.videoSpanId + ' .focusindicator').remove();
+    var element = $('#' + this.videoSpanId + ' .focusindicator');
+    UIUtil.destroyTooltip(element);
```

Comment:
We should try to do a global DOM listener and when an element is hidden or removed from the DOM we check if there is a tooltip attached and we destroy it.